### PR TITLE
Fix Jest issues

### PR DESCRIPTION
**Pull request by FeimeiChen**
# Ticket Link

[Ticket 1586](https://uhawaii.atlassian.net/browse/GROUPINGS-1586)

# List of squashed commits

- Commit 1: Fix Jest issues

# Test Checklist

- [x] Jest Tests Passed:
- [x] General Visual Inspection:

### CODE REVIEW

**Comment by FeimeiChen** [comment]
image
the result html page look like

File: app/jest.config.js [hunk 1]
```diff
@@ -6,8 +6,10 @@ module.exports = {
     rootDir: './',
     collectCoverageFrom: [
         './**/*.ts*',
+        '!./**/cypress/**/*.ts*',
+        '!./**/tests/**/*.ts*',
```
Comment (JorWo) [inline app/jest.config.js:10]: Could you use this instead? Thanks.
```
"coveragePathIgnorePatterns" : []
```

Comment (FeimeiChen) [reply app/jest.config.js]: done

**Review by JorWo — APPROVED** [review]
Thanks for fixing it up!

**Pull request merged**